test/fluree/db/query/misc_queries_test.clj

```diff
@@ -33,47 +33,132 @@
+                [:ex/User :id "http://example.org/ns/User"]
+                [:ex/User :rdf/type :rdfs/Class]
+                [:rdfs/Class :id "http://www.w3.org/2000/01/rdf-schema#Class"]
+                [:rdf/type :id "http://www.w3.org/1999/02/22-rdf-syntax-ns#type"]
+                [:f/t :id "https://ns.flur.ee/ledger#t"]
+                [:f/size :id "https://ns.flur.ee/ledger#size"]
+                [:f/flakes :id "https://ns.flur.ee/ledger#flakes"]
+                [:f/context :id "https://ns.flur.ee/ledger#context"]
+                [:f/branch :id "https://ns.flur.ee/ledger#branch"]
+                [:f/alias :id "https://ns.flur.ee/ledger#alias"]
+                [:f/data :id "https://ns.flur.ee/ledger#data"]
+                [:f/address :id "https://ns.flur.ee/ledger#address"]
+                [:f/v :id "https://ns.flur.ee/ledger#v"]
+                ["https://www.w3.org/2018/credentials#issuer"
+                 :id
+                 "https://www.w3.org/2018/credentials#issuer"]
+                [:f/time :id "https://ns.flur.ee/ledger#time"]
+                [:f/message :id "https://ns.flur.ee/ledger#message"]
+                [:f/previous :id "https://ns.flur.ee/ledger#previous"]
+                [:id :id "@id"]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 :id
+                 "fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 :f/time
+                 720000]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 "https://www.w3.org/2018/credentials#issuer"
+                 "did:fluree:TfCzWTrXqF16hvKGjcYiLxRoYJ1B8a6UMH6"]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 :f/v
+                 0]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 :f/address
+                 "fluree:memory://aac96d4d42bfecff44a7479259a1c8f3ccecca4df3cf42d177b84f7619b0baae"]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 :f/data
+                 "fluree:db:sha256:bbvrvnmzcotbulq3zo7cdakl3vacysgap6il37m4widvdp7vr353a"]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 :f/alias
+                 "query/everything"]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 :f/branch
+                 "main"]
+                ["fluree:commit:sha256:bs3bcu3rhzlidsdvg3jgvbplhzxxlevvhipxsnqvfk7em4juu2os"
+                 :f/context
+                 "fluree:memory:///contexts/b6dcf8968183239ecc7a664025f247de5b7859ac18cdeaace89aafc421eeddee"]]
```

Comment:
You can see we back a lot of fluree-generated commit data now if we pull the whole db after committing. This wasn't happening before because we had a caching bug that kept the commit flakes from being discoverable at all, even though they were there (see https://github.com/fluree/db/issues/333#issuecomment-1402068872 for discussion).